src/core/ExtensionBuilder.ts

```diff
@@ -10,17 +10,17 @@ import type {
     ExtensionDeps,
     ExtensionSpec,
     ExtensionWithOptions,
-    YEMarkSpec,
-    YENodeSpec,
+    WEMarkSpec,
+    WENodeSpec,
 } from './types/extension';
 import type {Keymap} from './types/keymap';
 
 type InputRulesConfig = Parameters<typeof inputRules>[0];
 type ExtensionWithParams = (builder: ExtensionBuilder, ...params: any[]) => void;
 
 type ConfigureMdCallback = (md: MarkdownIt) => MarkdownIt;
-type AddPmNodeCallback = () => YENodeSpec;
-type AddPmMarkCallback = () => YEMarkSpec;
+type AddPmNodeCallback = () => WENodeSpec;
+type AddPmMarkCallback = () => WEMarkSpec;
```

Comment:
nit:
Maybe we should give up using single-letter abbreviations and instead use full words?
For example:
`WysiwygEditorNodeSpec`

Reply:
Renamed to `ExtensionNodeSpec` and `ExtensionMarkSpec`